docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md

```diff
@@ -161,6 +162,20 @@ You can test against this setup by running the end-to-end tests ([end_to_end_tes
 
 The default configuration of the project uses a Stellar network observer to identify incoming Stellar payments. In case the Anchor relies on Circle, it should configure the project to use the Circle Payment Observer. For more information on how to do that, please refer to the [01.B - Circle Payment Observer](/docs/01%20-%20Running%20%26%20Configuring%20the%20Application/B%20-%20Circle%20Payment%20Observer.md) section.
 
+
+## Metrics
+The Anchor Platform exposes a Prometheus metrics endpoint at `<host>:8082/actuator/prometheus`. All standard Spring 
```

Comment:
still dependent on https://github.com/stellar/java-stellar-anchor-sdk/issues/297

Reply:
The issue with hardcoding is that some partners could have another service in this same port. If we could have this config in a yaml file, this would be great.

Reply:
@marcelosalloum 
due to #297, the port config has to be specified in the code - ex: 
      builder.properties(
          "management.endpoints.web.exposure.include=health,info,prometheus",
          "management.server.port=8082");

this would be configurable when #297 is resolved and we are able to specify this config in the config file

Reply:
Alright. I noticed that issue was not on the board yet, so we were not considering it in the sprint planning. Perhaps it can be tackled as part of the config refactor.